web/src/components/DandisetList.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <v-row>
+    <v-list
+      two-line
+      subheader
+    >
+      <v-list-item
+        v-for="(item, i) in items"
+        :key="i"
```

Comment:
```suggestion
        v-for="item in items"
        :key="item._id"
```
The id is guaranteed to deterministically map to the same element after any changes to the data. The position `i` may not always deterministically yield the same element. See: https://vuejs.org/v2/api/#key